tests/test_postgres.py

```diff
@@ -116,7 +116,8 @@ def test_cand_gen_cascading_delete(database_session):
 
     # Test that deletion of a Candidate does not delete the Mention
     x = session.query(PartTemp).first()
-    session.query(PartTemp).filter_by(id=x.id).delete(synchronize_session="fetch")
+    d = session.query(PartTemp).filter_by(id=x.id).first()
```

Comment:
nit: can we call this variable `candidate` or `cand` rather than `d`?

Reply:
I think `candidate` is better than `d`. I correct it.
As a side note, I was intended that `d` denotes `delete row` of `PartTemp` table. However, this `d` also means `delete candidate`. So, I update `d` to `candidate`. Thank you for your suggestion!